server/api/api/endpoints/model_endpoints.py

```diff
@@ -461,11 +478,30 @@ async def get_model_endpoint_monitoring_metrics_values(
 
     :returns:      A list of the results values for this model endpoint.
     """
-    return await run_in_threadpool(
+
+    read_data_coroutine = run_in_threadpool(
         mlrun.model_monitoring.db.v3io_tsdb_reader.read_data,
         project=params.project,
         endpoint_id=params.endpoint_id,
         metrics=params.metrics,
         start=params.start,
         end=params.end,
     )
+
+    if any([metric.name == "invocations" for metric in params.metrics]):
+        data, predictions = asyncio.gather(
+            read_data_coroutine,
+            run_in_threadpool(
+                mlrun.model_monitoring.db.v3io_tsdb_reader.read_predictions,
+                project=params.project,
+                endpoint_id=params.endpoint_id,
+                start=str(params.start.timestamp()),
+                end=str(params.end.timestamp()),
+                aggregation_window=params.aggregation_window or "1m",
```

Comment:
This constant should be more visible. Please add a constant for it in this module.